api.md

```diff
@@ -0,0 +1,115 @@
+# cozysdk-client
+
+SDK for serverless cozy apps
+
+## What is it for ?
+
+cozysdk-client is a native library that enables applications without a server to do requests directly, and simply to the data-system. 
+
+## What do we have inside ?
+
+To work with this library, you'll need to use the following functions :
+
+### create(docType, attributes)
```

Comment:
The `callback` is not showed in the signature. Is it on purpose? I think we should add it.

For example, on nodejs API docs, the callback is shown.